addons/mass_mailing/views/mass_mailing_views.xml

```diff
@@ -10,72 +10,6 @@
             parent="mass_mailing_menu_root" sequence="100"/>
 
         <!--  MAILING CONTACT -->
-        <record model="ir.ui.view" id="mail_mass_mailing_list_contact_rel_view_form">
```

Comment:
Same remark as removing the method: we should not remove views in stable as it may break addons inheriting from it. Removing should be done in master if necessary.